packages/dotcom-build-sass/src/index.ts

```diff
@@ -4,15 +4,18 @@ import type webpack from 'webpack'
 
 export type TPluginOptions = {
   includePaths?: string[]
+  prependData?: string
```

Comment:
Is this a valid option for all plugins or only the sass plugin? It might be better to set up the sass plugin to extend this generic Type with the new property rather than amending it here.

Reply:
> Is this a valid option for all plugins or only the sass plugin?
Only the sass plugin.

> It might be better to set up the sass plugin to extend this generic Type with the new property rather than amending it here.
I'm unclear on what you mean - could you please give code examples?